front_end/entrypoints/inspector_main/CohtmlPanel.ts

```diff
@@ -292,65 +347,24 @@ export class CohtmlPanelView extends UI.Widget.VBox implements SDK.TargetManager
 
       buttonWrapper.createChild('div', 'icon');
 
-      entryButton.onclick = async (event: Event) => {
-        let inputElement = (entryInput as HTMLInputElement);
-        if (inputElement.value.length == 0) {
-          return;
-        }
-        let newSize = Number(inputElement.value);
-        if (!isNaN(newSize)) {
-
-          // JSON specifies only 32-bit integers so we can't send a bigger
-          // number than 2^32 bytes (~ 2147 MBs; this should be enough for all caches)
-          const MAX_32BIT_INTEGER = 2147483647;
-          newSize = clamp(newSize, 0, MAX_32BIT_INTEGER);
-
-          switch (unitSelect.value) {
-            case "Count": break;
-            case "Bytes": break;
-            case "KBs": newSize *= 1024; break;
-            case "MBs": newSize *= 1024 * 1024; break;
-          }
-
-          let setCount = unitSelect.value == "Count";
-
-          // create the object that will be send to the backend
-          let state: Protocol.CohtmlDebug.RenoirCache = {
-            type: id,
-            capacityBytes: !setCount ? newSize : -1,
-            capacityCount: setCount ? newSize : -1
-          };
-
-          const button = event.currentTarget;
-          this.toggleButtonLoadingIndicator(button, true);
-          // send the data to the backend (the C++ code)
-          await this.cohtmlDebugModel?.setRenoirCacheState(state);
-
-          // Cohtml does not update the caches immediatly but rather on the
-          // next frame. Hence we can't request the new cache state immediatly but
-          // have to "wait a little bit". 100ms is plenty of time but still keeps the
-          // updating feeling interactive
-          setTimeout(async () => {
-            await this.fetchCacheStates();
-            this.toggleButtonLoadingIndicator(button, false);
-          }, 100);
-        }
-      };
-    }
+      if (unitSelect && id === 1) {
```

Comment:
can we add comment here about this `id == 1` and reference [this](https://github.com/CoherentLabs/cohtml/blob/develop/src/Inspector/CohtmlDebugAgent.h#L101) array.